app/assets/javascripts/backbone/views/breakeven_product_total_view.coffee

```diff
@@ -0,0 +1,104 @@
+class MS.Views.BreakevenProductTotalView extends Backbone.View
+
+  # The view is called from the editiable table view.
+
+  # events:
+  #   'click tr [data-action="delete"]': 'removeRow'
+  #   'click .actions [data-action="add"]': 'addRow'
+
+  initialize: (options) ->
+    @products = options.products
+
+    @totalFixedCosts = parseFloat(@$('.total_fixed_costs').val())
+
+    _.each @products, (product) =>
+      product.on "product:changed", () =>
```

Comment:
I guess I've never tried this particular thing, but you should be able to trigger the event on the DOM object associated with the Product View. All Backbone views have DOM elements. So if `myview.trigger('foo')` doesn't trigger an event on the DOM element (not sure, it might or might not), then `myview.el.trigger('foo')` definitely should. Then you can make use of the DOM's built-in event bubbling. In your event hash, assuming the listening view is attached to an element that wrapps the Product elements, you should just need to do e.g. `'product:changed': 'myfunc'`.